src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/ConsulWatcher.java

```diff
@@ -151,8 +151,8 @@ public Builder withBackoff(int initialReconnectBackoffMillis, int maxReconnectBa
             return this;
         }
 
-        public Builder requireDefaultConsistency() {
-            this.allowStale = false;
+        public Builder allowStale() {
```

Comment:
Accepting a boolean parameter would allow easier configuration:

```
boolean allowStale = config.getAllowStale();
builder.allowStale(allowStale);
```

vs

```
boolean allowStale = config.getAllowStale();
if (allowStale) {
    builder.allowStale();
}
```

Reply:
agreed